creatorkit/Toolbox.md

```diff
@@ -138,17 +132,30 @@ Initially you need to install dependencies required to build. run the following
 
 If you do not have repo installed run the following
 
+<code>$ mkdir -p ~/bin</code>
+
 <code>$ curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo</code>
 
 <code>$ chmod a+x ~/bin/repo</code>
 
+Update ~/.bashrc to add repo path in linux path permanently
+
+<code>export PATH=$PATH:~/bin/repo</code>
 
-You should now be able to run the following in a directory of your choice.
+You should now be able to run the following in a directory of your choice to get the code.
 
-<code>$ repo init -u  https://github.com/CreatorKit/manifest.git</code>
+<code>$ repo init -u  https://github.com/CreatorKit/manifest.git -b dev</code>
```

Comment:
I think the branch name can change as per user's choice, so the instruction should use a generic name "-b branch-name"